data/templates/css/wikimedia.scss

```diff
@@ -431,15 +496,48 @@ html {
             clear: right;
             float: right;
             width: $in-cols * 11 !important;
+            margin-left: 20px;
         }
     }
 }
 
-/* Composite resolutions */
+/////
+// Composite Resolutions
+/////
+
+@media (max-width: 720px) {
+    .infobox, 
+    .infobox_v2 {
+        width: 100% !important;
+    }
+}
+
 @media (device-width: 720px) and (device-height: 480px),
        (device-width: 720px) and (device-height: 576px) {
-    .mw-body {
-        font-family: $font1-composite;
-        font-size: 19.4px;  /* 22px * 88% because spec was at 110% scale not 125% */
+    #inside-content {
+        font-family: $font-primary-composite;
+
+        h2,
+        .firstHeading,
+        .eos-article-title {
+            border-bottom: 1px solid $divider-color-composite;
+        }
+
+        .disclaimer {
+            border-top: 1px solid $divider-color-composite;
+        }
+    }
+
+    .infobox, 
+    .infobox_v2,
+    .gallerytext,
+    .wikitable,
+    tbody,
+    .thumbcaption,
+    .disclaimer {
+        font-size: 1.0rem !important;  // 20px // Root: 20px
     }
 }
+
+
+
```

Comment:
So this is actually a separate problem, and I've noticed that too.

Here's what happened: I made a design decision that, in composite mode, adds a `border-bottom` to the `h2` and `eos-article-title` elements, b/c it's very hard to differentiate between sections when the screen is that small.

BUT for some reason, maybe in the ingestor, when a section is hidden (like the "References" section or something), it seems that the `h2` element is still there in the HTML, even though it doesn't display anything. That's why the `border-bottom` is still applied.

I'm not sure if that's something that can be fixed with CSS, but I guess for now, since it's making that ugly bug, I will take out the `border-bottom` until I come up with a better solution.